adapters/handlers/rest/handlers_schema.go

```diff
@@ -47,8 +47,6 @@ func (s *schemaHandlers) addClass(params schema.SchemaObjectsCreateParams,
 		}
 	}
 
-	// TODO add all permissions in RBAC for the logged in user to this class
```

Comment:
**note:** autoScehma is tested and it uses `GetCachedClass()` which alread does authZ